app/src/main/res/layout/layer_view.xml

```diff
@@ -14,7 +14,7 @@
         android:id="@+id/layerName"
```

Comment:
Your layouts are missing the xml tagline at the top that is supposed to be there. This is causing your toolbar xml not to even show in Github.

Reply:
added the tag needed